lib/routes/account.js

```diff
@@ -1420,6 +1420,77 @@ module.exports = function (
       }
     },
     {
+      method: 'GET',
+      path: '/account/sessions',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        response: {
+          schema: isA.array().items(isA.object({
+            id: isA.string().regex(HEX_STRING).required(),
+            lastAccessTime: isA.number().min(0).required().allow(null),
+            lastAccessTimeFormatted: isA.string().optional().allow(''),
+            userAgent: isA.string().max(255).required().allow(''),
+
```

Comment:
Were the spaces to separate response visually? I think it might be ok to not have spaces because the device properties are prefixed with `device`.